tests/resources/insecurity/test_food_insecurity_csv.csv

```diff
@@ -2,3 +2,7 @@
 2020 Food Insecurity  %",2018 Child Food Insecurity  %,"[Revised Projections – Oct 2020] 
 2020 Child Food Insecurity  %"
 "Jo Daviess County, Illinois",0.09,0.133,0.128,0.212
```

Comment:
Are we rounding these to thousandths?

Is that what design wants?

Reply:
This data is an excerpt from the source data excel file. Are you asking about the source data or the outputed bins?

Reply:
Just want to make sure it wasn't being rounded, because it's not very precise as a percentage